multi-era/rust/Cargo.toml

```diff
@@ -23,8 +26,13 @@ derivative = "2.2.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.57"
 schemars = "0.8.8"
-wasm-bindgen = { version = "0.2.87" }
 
 # only for declaring hash types
 bech32 = "0.7.2"
 hex = "0.4.0"
+
+# non-wasm
+noop_proc_macro = { version = "0.3.0" }
```

Comment:
I had this as `optional = true` but for some reason when consuming it from dcspark-core/carp it would not be able to find the dependancy. Strangely having it as optional worked fine for the `cml-chain` dependency. Both were specified in the same way in the same `Cargo.toml` file in dcspark-core so I have  no idea why this is the case.

Reply:
doesn't `cml-chain` have it as `optional = false`? which would be the same as not writing it

Reply:
Oops, I saw it specified and I guess my brain just read it as `= true` automatically. That makes sense now.